x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/date/DateDiffTests.java

```diff
@@ -71,48 +71,6 @@ public static Iterable<Object[]> parameters() {
                 )
             )
         );
-        suppliers.add(
```

Comment:
It looks like these are covered by the "WithDefaultChecks" with that switch statement getting the expected error messages. Right?

Reply:
Yes! Those are wrong-type cases, so they are covered already in the autogenerated ones